PWGHF/HFC/TableProducer/derivedDataCreatorCorrelationsReduced.cxx

```diff
@@ -347,13 +349,20 @@ struct HfDerivedDataCreatorCorrelationsReduced {
         registry.fill(HIST("hDcaZVsPtAssoc"), assTrk.dcaZ(), assTrkPt);
 
         double deltaPhi = RecoDecay::constrainAngle(assTrk.phi() - trigCand.phi(), -o2::constants::math::PIHalf);
-        rowSEPairs(rowCollisions.lastIndex(), trigCandPt, assTrkPt, deltaEta, deltaPhi);
         rowAssocTrkSels(assTrk.tpcNClsCrossedRows(), assTrk.itsClusterMap(), assTrk.itsNCls(), assTrk.dcaXY(), assTrk.dcaZ());
         if constexpr (candType == CandType::Hadron) {
```

Comment:
I think you can move this block of code out of the associated track loop, since it's about the trigger. 
In this way, you can avoid the `bool isFirstAssoc{true};` and its check within this loop